src/v/cluster/cluster_discovery.h

```diff
@@ -111,59 +111,28 @@ class cluster_discovery {
     // returns true.
     ss::future<bool> dispatch_node_uuid_registration_to_seeds(model::node_id&);
 
+    // Requests `cluster_bootstrap_info` from the given address, returning
+    // early with a bogus result if it's already been determined if this node
+    // is a cluster founder.
     ss::future<cluster_bootstrap_info_reply>
-      request_cluster_bootstrap_info_single(net::unresolved_address) const;
+    request_cluster_bootstrap_info_single(const net::unresolved_address&) const;
 
-    /**
-     * If not already present, requests cluster_bootstrap_info from all seeds
-     * and stores it into _cluster_bootstrap_info_replies
-     */
+    // Requests cluster_bootstrap_info from all seeds to determine whether the
+    // local node is a cluster founder, and if so, populates
+    // `_founding_brokers`. Validates that all seeds are consistent with one
+    // another and agree on the set of founding brokers.
+    //
+    // Sets `_is_cluster_founder` upon completion.
     ss::future<> fetch_cluster_bootstrap_info();
```

Comment:
A better name now can be `init_founding_brokers` / `discover_founding_brokers` / `discover_cluster_founders` / something alike

Reply:
SGTM, went with `discover_founding_brokers()`